code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -29,34 +29,41 @@
 	switch(act)
 		//Cooldown-inducing emotes
 		if("ping", "pings", "buzz", "buzzes", "beep", "beeps", "yes", "no", "buzz2")
-			if(species.name == "Machine")		//Only Machines can beep, ping, and buzz, yes, no, and make a silly sad trombone noise.
+			var/found_machine_head = FALSE
+			if(ismachine(src))		//Only Machines can beep, ping, and buzz, yes, no, and make a silly sad trombone noise.
```

Comment:
oh, you don't set `found_machine_head` here - hence IPCs not being able to use their emotes

Reply:
Yep. Fixed.